src/main/java/com/slack/kaldb/server/KaldbIndexer.java

```diff
@@ -167,7 +132,14 @@ public void close() {
       LOG.warn("Failed to close kafka consumer cleanly because of an exception.", e);
     }
 
-    chunkManager.close();
+    chunkManager.stopAsync();
```

Comment:
nice! So this is where guava helps with shutdown sequencing - i.e before the indexer shuts down first ensure kafka is shutdown and the chunk has been written out

Reply:
I guess we might face a regular pod shutdown issue and if it's during writing a chunk to S3 15 seconds here might not be enough? Something for later I guess

Reply:
Yeah, the timing is something we can definitely tweak if we need to - the 15s limit is mostly because I have the overall shutdown limited to 30s currently.